solr/core/src/java/org/apache/solr/cloud/api/collections/TimeRoutedAlias.java

```diff
@@ -141,6 +145,9 @@ public TimeRoutedAlias(String aliasName, Map<String, String> aliasMetadata) {
 
     //optional:
     maxFutureMs = params.getLong(ROUTER_MAX_FUTURE, TimeUnit.MINUTES.toMillis(10));
+    // the date math configured is an interval to be subtracted from the most recent collection's time stamp
+    preemptiveCreateMath = params.get(ROUTER_PREEMPTIVE_CREATE_MATH) != null ?
```

Comment:
Perhaps this should detect if it already starts with a plus or minus and if so removes it, thus making it more friendly to people who like to include the operator?